.travis.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         - sudo mv docker-compose /usr/local/bin
         - sudo service mysql stop
       install:
-        - python3.6 codalab_service.py build services --version ${TRAVIS_BRANCH} --pull --push
+        - python3.6 codalab_service.py build services --version ${TRAVIS_BRANCH} --pull $([ -z "${CODALAB_DOCKER_USERNAME}" ] || echo "--push")
```

Comment:
Do we need this to be python3.6?  We were trying to replace everything with python3.